react/Button/Button.less

```diff
@@ -17,6 +17,8 @@
 
 .root {
   .touchableText();
+  display: block;
```

Comment:
IIRC buttons are `inline-block` by default. I'm guessing we should probably change this?